config/locales/en.yml

```
@@ -1245,7 +1245,7 @@ en:
         api_backend_https: "Private address of your API that will be called by the API gateway."
         api_backend_http: |
           Private address of your API that will be called by the API gateway.
-          <span class="warn">For end-to-end encryption your private base URL scheme should be https</span>.
+          <span class="protocol-warn">For end-to-end encryption your private base URL scheme should be https.</span>
```

Hmm.... We are going to support secure websocket so this might change, maybe:

```suggestion
          <span class="protocol-warn">For end-to-end encryption your private base URL should use a secure protocol..</span>
```

---

I think we should change the message until websockets are supported

---

https://github.com/3scale/porta/pull/1527